components/service/experiments/README.md

```diff
@@ -64,14 +69,370 @@ For any technical tests, we do have a Kinto dev server available, which can be f
+        }
+      },
+      "match": {
+        "title": "Matching",
+        "type": "object",
+        "description": "A list of optional matchers, which allow restricting the experiment to e.g. specific application ids.",
+        "properties": {
+          "app_id": {
+            "type": "string",
+            "description": "Match specific application ids. A regex. E.g.: ^org.mozilla.fennec|org.mozilla.firefox_beta|org.mozilla.firefox$",
+            "minLength": 1,
+            "maxLength": 1000
+          },
+          "app_display_version": {
+            "description": "The application's version number. A regex. E.g.: '47.0a1', '46.0'",
+            "type": "string"
+          },
+          "app_min_version": {
+            "description": "The application's minimum version number. E.g.: '47.0.11', '46.0'",
+            "type": "string"
+          },
+          "app_max_version": {
+            "description": "The application's maximum version number. E.g.: '47.0.11', '46.0'",
+            "type": "string"
+          },
+          "locale_country": {
+            "description": "Match country, pulled from the default locale. A regex. E.g.: USA|ITA",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 1000
+          },
+          "locale_language": {
+            "description": "Language, pulled from the default locale. A regex. E.g.: eng|esp",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 1000
+          },
+          "device_model": {
+            "description": "Device name. A regex.",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 1000
+          },
+          "device_manufacturer": {
+            "description": "Device manufacturer",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 1000
+          },
+          "regions": {
+            "default": [],
+            "description": "Compared with GeoIP lookup, where supported.",
+            "items": {
+              "default": "",
+              "description": "Similar to a GeoIP lookup",
+              "minLength": 1,
+              "maxLength": 1000,
+              "title": "Regions",
+              "type": "string"
+            },
+            "title": "Regions",
+            "type": "array",
+            "uniqueItems": true
+          },
+          "debug_tags": {
+            "default": [],
+            "description": "Target specific debug tags only. This allows testing of experiments for only specific active users for QA etc.",
+            "items": {
+              "default": "",
+              "description": "A debug tag set through the libraries debug activity.",
+              "minLength": 1,
+              "title": "Debug tag",
+              "type": "string"
+            },
+            "title": "Debug tags",
+            "type": "array",
+            "uniqueItems": true
+          }
+        }
+      }
+    }
+  }
+  ```
```

Comment:
What is the reason for inlining this here?  It might be hard to remember to keep it in sync (and I also think a casual reader of this document may not care in most cases).

Reply:
I was trying to remove the dependency to a gist in my personal github (via the link to the schema).  It started off with Georg's gist, which was forked so I could make changes.  I didn't like having all the moving parts in separate repos, which I thought would be even more difficult to keep in sync.

Reply:
I suppose I could create separate README's and just link them in the main one here...

Reply:
Ah -- I didn't notice it was linking to a gist.  What if we add a jsonschema directory right here (in android components) put these as `.json` files there and link to that?

Reply:
I think I'd rather just leave them as inline in a separate file so that there can be headings, and explanations, in case someone were to land on just that file.